src/components/pages/InstructorAddCourse/InstructorAddCourseFormSchema.js

```diff
@@ -0,0 +1,56 @@
+import * as yup from 'yup';
+
+const InstructorAddCourseFormSchema = yup.object().shape({
+  course_type_id: yup.string().required('Please select a course'),
+  day: yup
+    .string()
+    .oneOf([
+      'Monday',
+      'Tuesday',
+      'Wednesday',
+      'Thursday',
+      'Friday',
+      'Saturday',
+      'Sunday',
+    ])
+    .required('Please select a day'),
+  size: yup
+    .number()
+    .integer('Number must be a whole value')
+    .min(2, 'Class size must have at least 2 students')
+    .required('You must specify a class size'),
+  min_age: yup
+    .number()
+    .lessThan(18, 'Must be 17 years or younger')
+    .moreThan(6, 'Must be 7 years or older')
+    .required('You must specify a minimum age'),
+  max_age: yup
+    .number()
+    .lessThan(18, 'Must be 17 years or younger')
+    .moreThan(6, `Must be 7 years or older`)
+    // .moreThan(min_age, `Must be older than ${min_age}`)
```

Comment:
Is this comment necessary?

Reply:
Yeah i will take this one out. I just had it as a reference